assets/js/editor.js

```diff
@@ -103,15 +103,15 @@ $gp.editor = (
 				} );
 			},
 			keydown: function( e ) {
-				var target, container;
+				var target, container, original, approve, reject;
 
-				if ( 27 === e.keyCode ) {
+				if ( 27 === e.keyCode ) { // Escape = Next editor.
 					$gp.editor.hide();
-				} else if ( 33 === e.keyCode ) {
+				} else if ( 33 === e.keyCode || ( 38 === e.keyCode && e.ctrlKey ) ) { // Page Down or Ctrl-Up Arrow = Previous editor.
```

Comment:
CTRL-[UP/DOWN/LEFT/RIGHT] are used by OS X to open Mission Control and to change spaces.